hack/local-development/local-garden/run-kube-apiserver

```diff
@@ -67,4 +67,5 @@ docker run -d --name kube-apiserver -l $LABEL $ADD_HOSTS --network gardener-dev
   --service-account-signing-key-file="/keys/sa.key" \
   --service-account-issuer="local-garden" \
   --service-cluster-ip-range="10.0.0.0/24" \
-  --secure-port=$LISTEN_PORT
+  --secure-port=$LISTEN_PORT \
+  --disable-admission-plugins="ValidatingAdmissionWebhook"
```

Comment:
Why did you add this?

Reply:
Added by accident with the last force push :/.